localstack/utils/aws/aws_models.py

```diff
@@ -184,6 +189,58 @@ def set_dead_letter_config(self, data):
         if ':sqs:' not in target_arn and ':sns:' not in target_arn:
             raise Exception('Dead letter queue ARN "%s" requires a valid SQS queue or SNS topic' % target_arn)
 
+    def get_function_event_invoke_config(self):
+        return {
+            'LastModified': str(self.last_modified),
+            'FunctionArn': str(self.id),
+            'MaximumRetryAttempts': self.max_retry_attempt,
+            'MaximumEventAgeInSeconds': self.max_event_age,
+            'DestinationConfig': {
+                'OnSuccess': {
+                    'Destination': str(self.on_successful_invocation)
```

Comment:
Perhaps we should only set the `Destination` attribute in lines 200 and 203 if the respective fields `on_successful_invocation`/`dead_letter_config` are configured (i.e., not empty).